nix/haskell.nix

```diff
@@ -147,10 +147,6 @@ haskell-nix: haskell-nix.cabalProject' [
         ] ++ lib.filter
           (drv: lib.isDerivation drv && drv.name != "regenerate-materialized-nix")
           (lib.attrValues haskell-build-tools));
-
-        CARDANO_NODE_CONFIGS = pkgs.cardano-node-deployments;
-
-        meta.platforms = lib.platforms.unix;
```

Comment:
Not sure what it did, but nothing's broken yet, so hope it was unneeded.

Reply:
Or we do have a hydra failure which might be caused by this:

```
unpacking sources
unpacking source archive /nix/store/bqmqf7dxn830zzyavpr0wy1crwjvwfnx-cardano-wallet-src-root-lib-wallet-test-unit-root
source root is cardano-wallet-src-root-lib-wallet-test-unit-root
patching sources
building
/nix/store/wfavk9lnf9dkd1j9g1jgp1c513dzrkvw-stdenv-linux/setup: line 1401: /nix/store/blj5x8ssrq74d208x9ffdpx2y105i7wn-cardano-wallet-test-unit-x86_64-unknown-linux-musl-2022.10.6/bin/unit: cannot execute binary file: Exec format error
```

Reply:
But I wonder if that is not just some kind of hydra flakiness 🤔 *attempting to retry*

Reply:
Succeeded here: https://hydra.iohk.io/build/20306647